### PR TITLE
Update compiler_builtins to 0.1.7 to get windows/arm fix

### PR DESCRIPTION
**Pull request by jordanrh1**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -2092,7 +2092,7 @@ name = "rand_chacha"
 version = "0.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "rand_core 0.3.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "rand_core 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
```
Comment (sanxiyn) [inline Cargo.lock]: Is this correct?

Comment (jordanrh1) [reply Cargo.lock]: No, that's weird. Not sure how that got there. Will fix.

**Comment by petrochenkov** [comment]
@jordanrh1 
This needs a rebase.

**Comment by jordanrh1** [comment]
Rebased.

**Comment by mati865** [comment]
This PR requires rebase, instructions are in https://github.com/rust-lang/rust/pull/58789#issuecomment-473457563

**Comment by Centril** [comment]
Ping from triage, @jordanrh1, you have some rebasing to do.

**Comment by mati865** [comment]
Superseded by https://github.com/rust-lang/rust/pull/59643

**Comment by sanxiyn** [comment]
#59643, which updated compiler_builtins to 0.1.9, landed. Closing.

**Pull request closed without merging**